x-pack/plugins/fleet/public/hooks/use_kibana_link.ts

```diff
@@ -4,12 +4,62 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
+import type { HttpStart } from 'src/core/public';
+
+import { KibanaAssetType } from '../types';
 
 import { useStartServices } from './';
 
 const KIBANA_BASE_PATH = '/app/kibana';
 
+const getKibanaLink = (http: HttpStart, path: string) => {
+  return http.basePath.prepend(`${KIBANA_BASE_PATH}#${path}`);
+};
+
+/**
```

Comment:
Appreciate the callouts here on these stopgap solutions 👍

Reply:
I've created this issue for visibility, https://github.com/elastic/kibana/issues/102740